tests/unit/acl-v2.tcl

```diff
@@ -75,11 +75,11 @@ start_server {tags {"acl external:skip"}} {
 
         # Test invalid selector syntax
         catch {r ACL SETUSER invalid-selector " () "} err
-        assert_match "*ERR*Syntax error*" $err
+        assert_match "ERR*Syntax error*" $err
         catch {r ACL SETUSER invalid-selector (} err
-        assert_match "*Unmatched parenthesis*" $err
+        assert_match "ERR*Unmatched parenthesis*" $err
         catch {r ACL SETUSER invalid-selector )} err
-        assert_match "*ERR*Syntax error" $err
```

Comment:
What I actually want to change is this line:
1. this line work as the same as line 78 but without the `*`
2. without the `*`, test fails when extending message to expose more info (internal fork)